web/README.md

```diff
@@ -21,8 +21,25 @@ The following folders contain the distribution for KeymanWeb:
 ## Usage
 Open **index.html** or **samples/index.html** in your browser. Be sure to compile KeymanWeb before viewing the pages.
 
+Refer to the samples for usage details.
+
 To view pages using compiled KeymanWeb,
 1. cd to **keyman/web/source**
 2. Run `/.build.sh`
 
-Refer to the samples for usage details.
+### `build.sh` options:
+
+*  `-ui`
+    - to compile desktop user interface modules to output folder
```

Comment:
Best viewed [in code mode](https://github.com/keymanapp/keyman/tree/web-no-minify-build-option/web).